disnake/types/guild.py

```diff
@@ -161,3 +161,25 @@ class RolePositionUpdate(TypedDict):
 
 class MFALevelUpdate(TypedDict):
     level: MFALevel
+
+
+class CreateGuildPlaceholderRole(CreateRole):
+    id: Snowflake
+
+
+class CreateGuildPlaceholderChannel(CreateGuildChannel, total=False):
+    id: Snowflake
```

Comment:
Why the `total=False`?

Reply:
`id` is optional, the endpoint doesn't require placeholder IDs to be specified. I've changed it to use `NotRequired` in https://github.com/DisnakeDev/disnake/pull/578/commits/4bc79c0ef9d78b5cea1d2595e1ec92acd28a95c6 to keep it consistent with the base type though.